### PR TITLE
Add HELPINFO and associated constants.

### PR DESCRIPTION
**Pull request by bradfitz**

### CODE REVIEW

File: help.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+)
+
+// HELPINFO is the structure sent on WM_HELP messages.
+//
+// See https://docs.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-helpinfo
+type HELPINFO struct {
+	Size        uint32 // cbSize, the struct size in bytes
+	ContextType int32  // either HELPINFO_WINDOW or HELPINFO_MENUITEM
+	CtrlId      int32
+	ItemHandle  HWND
```
Comment (dblohm7) [inline help.go]: Although the docs are terribly unclear about this, AIUI, `ItemHandle` could be either a `HWND` or a `HMENU`, so we should probably make this a generic `Handle` instead.

Comment (bradfitz) [reply help.go]: Done.

Comment (bradfitz) [reply help.go]: I'm going to assume that was your only comment and the rest is fine? Submitting, but happy to make more changes if needed.

Comment (dblohm7) [reply help.go]: Yep, we're good.

**Pull request merged**